GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/Podfile

```diff
@@ -1,18 +1,34 @@
 use_frameworks!
 
-target 'GDTCCTWatchOSIndependentTestAppWatchKitApp' do
+target 'GDTCCTWatchOSIndependentTestAppWatchKitExtension' do
   platform :watchos, '6.0'
 
+  # Pods for GDTCCTWatchOSIndependentTestAppWatchKitExtension
+  pod 'SwiftProtobuf', '~> 1.0'
+  pod 'GoogleDataTransport', :path => '../../'
+  pod 'GoogleDataTransportCCTSupport', :path => '../../'
+  pod 'FirebaseAnalyticsInterop', :path => '../../'
+  pod 'FirebaseAuthInterop', :path => '../../'
+  pod 'FirebaseCore', :path => '../../'
+  pod 'GoogleUtilities', :path => '../../'
+  pod 'FirebaseMessaging', :path => '../../'
+  pod 'FirebaseInstanceID', :path => '../../'
+  pod 'FirebaseCoreDiagnostics', :path => '../../'
+  pod 'FirebaseCoreDiagnosticsInterop', :path => '../../'
+  pod 'FirebaseInstallations', :path => '../../'
+  pod 'FirebaseStorage', :path => '../../'
+
 end
 
-target 'GDTCCTWatchOSIndependentTestAppWatchKitExtension' do
+target 'GDTCCTServiceExtension' do
   platform :watchos, '6.0'
```

Comment:
should this still be watchOS 6.0?

Reply:
UNNotificationServiceExtension is available on watchOS 6.0+. Do you think we shall not specify the watchOS platform or the version NO?

Reply:
I guess my question is: are we able to support watchOS versions < 6.0? Many extension-type apps are < 6.0.

Reply:
cc: @chliangGoogle  Not so far:

```
$ git grep watchos *.podspec
Firebase.podspec:    ss.watchos.deployment_target = '6.0'
Firebase.podspec:    ss.watchos.deployment_target = '6.0'
Firebase.podspec:    ss.watchos.deployment_target = '6.0'
FirebaseAnalyticsInterop.podspec:  s.watchos.deployment_target = '6.0'
FirebaseAuthInterop.podspec:  s.watchos.deployment_target = '6.0'
FirebaseCore.podspec:  s.watchos.deployment_target = '6.0'
FirebaseCoreDiagnostics.podspec:  s.watchos.deployment_target = '6.0'
FirebaseCoreDiagnosticsInterop.podspec:  s.watchos.deployment_target = '6.0'
FirebaseInstallations.podspec:  s.watchos.deployment_target = '6.0'
FirebaseInstanceID.podspec:  s.watchos.deployment_target = '6.0'
FirebaseMessaging.podspec:  s.watchos.deployment_target = '6.0'
FirebaseStorage.podspec:  s.watchos.deployment_target = '6.0'
FirebaseStorageSwift.podspec:  s.watchos.deployment_target = '6.0'
GoogleDataTransport.podspec:  s.watchos.deployment_target = '6.0'
GoogleDataTransportCCTSupport.podspec:  s.watchos.deployment_target = '6.0'
GoogleUtilities.podspec:  s.watchos.deployment_target = '6.0'
```

Reply:
the independent watch is also watchOS 6.0+. I guess it depends on which extension you try to support.

Reply:
depends on which extension you try to support +1
For watchOS, notification service extension 6.0+
Intents extension 3.2+